code/modules/mining/remote_fob.dm

```diff
@@ -0,0 +1,412 @@
+/////////For remote construction of FOB using a computer on the ship during setup phase
```

Comment:
If you get the time, its worth splitting this file into multiple for the eye / computer / actions.
The file is getting large enough that even git says its a large diff.